### ISSUE TITLE
Integrating magic wand tool / QuPath with DSA

### ISSUE THREAD

**Issue by yijiangchen** [DigitalSlideArchive/HistomicsTK#871]
Dear authors,

Thanks for creating such a great tool for the community. The tool provides a great platform for computational imaging engineers to collaborate with a board of pathologists. However, from our experience of working with a panel of pathologists, we got some feedback from them that, the polygon tool alone was not optimal in making accurate annotations. A large part of digital pathology community has been using Qupath for routine analyses and annotation making, including using tools such as magic wand, brush (or eraser) to fine-tune the annotations. That said, we were wondering if the authors can integrate tools such as magic wand into DSA, or even better, to allow user to upload Qupath annotations, and convert to json, to display annotations made in Qupath and allow user to make changes online.

Thanks

**Comment by choosehappy** [comment]
qupath and gimp both have open-source implementations and there is a JS version here: 
https://github.com/Tamersoul/magic-wand-js

with a live demo here:

https://jsfiddle.net/Tamersoul/dr7Dw/

that we've played with it and it works pretty well, but ended up removing in favor of our superpixel approach. that said, we may bring it back, and if we do we'll also start with this implementation

**Comment by cooperlab** [comment]
I agree this can be very useful and we will discuss this with the developers. Would want to think through how this meshes with the way we handle annotations.
